packages/client/src/apis/rewards/RewardsQueryAPI.ts

```diff
@@ -1,3 +1,115 @@
-import { BaseVertexAPI } from '../base';
+import { toBigInt } from '@vertex-protocol/utils';
+import { BaseRewardsAPI } from './BaseRewardsAPI';
+import { ClaimLiquidTokensParams, VrtxTokenAmountParams } from './types';
 
-export class RewardsQueryAPI extends BaseVertexAPI {}
+enum FunctionIndex {
```

Comment:
nit: this could be more generically named as `SatelliteTransactionType`.

If we don't need a 1-1 mapping of enum name to contract function name, it is usually convention to use `CAPITAL_SNAKE_CASE` (i.e `STAKE_AS`).

It is also [good practice](https://blog.logrocket.com/typescript-enums-vs-types/#typescript-enums-best-practices) to explicitly declare numerical values